common/tools/dev-tool/src/config/rollup.base.config.ts

```diff
@@ -15,6 +15,19 @@ interface PackageJson {
   devDependencies: Record<string, string>;
 }
 
+// core-tracing (because of opentelemetry) requires some symbols to be
+// exported.
+const openTelemetryNamedExports = {
```

Comment:
@witemple-msft, @xirzec  - I did...this and I'm wondering what you both think about it. 

`core-tracing` basically requires all the other projects to carry along some fairly boilerplate config for rollup. For some projects, like this [one](https://github.com/Azure/azure-sdk-for-js/pull/13665/files#diff-5b37b101df390fb580ac5d79217114fe5d2195a97b60d4e69d821c6d878640b2R11) I was able to just directly import a .js file that I put in the core-tracing package folder (since it's really core-tracing that "owns" this bit of rollup config)

(the 'shared' config file is here)
https://github.com/Azure/azure-sdk-for-js/pull/13665/files#diff-be2736da5e98f0601219a7118ef5b1e0a98cb4a49f4b1e32deaf79fc29311162

However, for ones that are using dev-tool's standardized config I didn't do that because I wanted a chance to talk about if this is the "right" pattern or if we prefer to just duplicate the config across projects and I figured it was easiest to put it into the code. :)

Reply:
The shared config is in stasis until we can move to rollup 2, currently blocked upstream. If these are the exports that we need for the new version of OT, then it's appropriate to add them to the shared rollup config.

Reply:
Having this centralized along with other rollup config would be great